### DISCUSSION TITLE
Upload form wireframes - second draft

### DISCUSSION THREAD

**Discussion by bjr70** [MESH-Research/knowledge-commons-repository#91] [Show and tell]
Note: This is not meant to convey look and feel. This is just a set of wireframes that is meant to convey the organization of information and where controls are on the page. 

https://miro.com/app/board/uXjVMH3IPzM=/?share_link_id=233490434312 

We are only looking for substantive feedback, not line edits. This is about the organization of information and the controls used to navigate and populate the form. 

Please submit any feedback by 12pm Thursday 5/25. Thanks!

**Comment by mikethicke** [comment]
Sorry to be late on this. Looks great overall. A couple of minor questions:

- Do we need to ask users if it is a metadata only deposit? Can't we just infer that if they don't attach any files?
- Do we need a radio button for having an existing DOI, or can we just have the input field and they can fill it in or not?
- I think I would call the continue button just "Continue" not "Save and Continue" as most likely we are either continuously saving as users enter data or not saving at all until final submission.

**Reply by bjr70** [reply]
Yeah I think we should be explicit about a metadata-only deposit. They may forget to attach files and I don't think we can infer that. 

Hmmm, Ian, do we need a radio button? We have the "was this deposit previously published" on the current repository, but I'm not sure it's necessary.

Using "continue" on the button is fine with me.

**Reply by monotasker** [reply]
What if, instead of a "metadata-only deposit" question at the beginning, we have an option on the final file-upload page of the form to check "no files to upload"? Would that be a bit more intuitive for users?

The radio button for the DOI field at present toggles a button that pre-mints a DOI for the record and shows it to the depositing user. So they can get a DOI from the upload form before they publish a draft. I believe it's only then registered with Datacite after the draft is published. (Not positive about the last part yet.)

"Continue" works for me too.

**Reply by mikethicke** [reply]
From talking about this with Ian recently:

I would suggest not having the question, and if a user submits a deposit with no file attachments, they are presented with a confirmation dialog making sure this isn't a mistake.
